plugins/tar/commands

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+set -eo pipefail; [[ $DOKKU_TRACE ]] && set -x
+source "$(dirname $0)/../common/functions"
+
+case "$1" in
+  tar-from|tar:from)
+    verify_app_name "$2"
+    APP=$2
+    URL=$3
+    shift 3
+    curl -# --insecure -L "$URL" | dokku tar-in $APP $@
```

Comment:
Should just be `tar:in` and we can remove the aliases.

The other thing is that - to reduce the `DOKKU_TRACE` output - the `tar:in` command can be extracted into it's own function and just executed instead of shelling out to dokku.